PGoApi/Classes/PtcOAuth.swift

```diff
@@ -67,13 +67,18 @@ public class PtcOAuth: PGoAuth {
                 let regex = try! NSRegularExpression(pattern: "access_token=([A-Za-z0-9\\-.]+)&expires=([0-9]+)", options: [])
                 let matches = regex.matchesInString(value, options: [], range: NSRange(location: 0, length: value.utf16.count))
                 
+                guard matches.count > 0 else {
```

Comment:
Wouldn't it be better to extract the first match and fail if it's nil? Then we wouldn't need for subscripting the array of matches later on.

```Swift
guard let match = matches.first else {
    self.delegate?.didNotReceiveAuth()
    return
}
```